near-sdk/src/collections/lookup_map.rs

```diff
@@ -284,4 +284,22 @@ mod tests {
             assert_eq!(map.get(&key).unwrap(), value);
         }
     }
+
+    #[test]
+    fn test_debug() {
+        let mut rng = rand_xorshift::XorShiftRng::seed_from_u64(4);
+        let mut map = LookupMap::new(b"m");
+        let mut baseline = vec![];
+        for _ in 0..10 {
+            let key = rng.gen::<u64>();
+            let value = rng.gen::<u64>();
+            baseline.push((key, value));
+            map.insert(&key, &value);
+        }
+
+        assert_eq!(
+            format!("{:?}", map),
+            format!("LookupMap {{ key_prefix: {:?}, el: PhantomData }}", map.key_prefix)
```

Comment:
This is maybe one you want to have a custom impl so you leave out the `PhantomData`

Reply:
I left it in because the existing Vector implementation had it in as well. I will delete it from both places then.

Reply:
yeah, feel free to update the old `Vector` to remove this, but it's not a big deal as this impl is being deprecated in favour of the one in `near_sdk::store`